examples/javascript/pkce-browser/index.html

```diff
@@ -0,0 +1,116 @@
+<!doctype html>
+<html>
+
```

Comment:
nit: Same here. Add extra lines in all the other examples or remove for consistency